maintenance_changing_maintainers.es.Rmd

```diff
@@ -1,59 +1,61 @@
-# Changing package maintainers {#changing-maintainers}
+# Cambio de mantenedores de paquetes {#changing-maintainers}
 
-```{block, type='summaryblock'}
-This chapter presents our guidance for taking over maintenance of a package.
+```{block, type="summaryblock"}
+Este capítulo presenta nuestra guía para asumir el mantenimiento de un paquete.
 ```
 
-## Do you want to give up maintenance of your package?
+## ¿Quieres dejar el mantenimiento de tu paquete?
 
-We have a call for contributors section in our newsletter that comes out every two weeks. The section is called _Call For Contributors_. In that section we highlight packages looking for new maintainers. If you are looking to leave the role of maintainer of your package, get in touch with us and we can highlight your package in our newsletter. We've been very successful thus far, finding new maintainers for six out of six packages.
+Tenemos una sección de petición de colaboradores en nuestro boletín que sale cada dos semanas. La sección se llama *Llamada a colaboradores*. En esa sección destacamos los paquetes que buscan nuevos mantenedores. Si quieres dejar el papel de mantenedor de tu paquete, ponte en contacto con nosotros y podremos destacar tu paquete en nuestro boletín. Hasta ahora hemos tenido mucho éxito, encontrando nuevos mantenedores para seis de los seis paquetes.
 
-## Do you want to take over maintenance of a package?
+## ¿Quieres encargarte del mantenimiento de un paquete?
 
-We have a call for contributors section in our newsletter that comes out every two weeks. The section is called _Call For Contributors_. In that section we highlight packages looking for new maintainers. If you are not subscribed to the newsletter already, it's a good idea to [subscribe](https://news.ropensci.org/) to get notified when there's a package looking for a new maintainer.
+Tenemos una sección de solicitud de colaboradores en nuestro boletín que sale cada dos semanas. La sección se llama *Llamada a colaboradores*. En esa sección destacamos los paquetes que buscan nuevos mantenedores. Si aún no estás suscrito al boletín, es una buena idea [suscribirte a](https://news.ropensci.org/) para recibir una notificación cuando haya un paquete que busque un nuevo mantenedor.
 
-## Taking over maintenance of a package
+## Asumir el mantenimiento de un paquete
 
-* Add yourself as the new maintainer in the DESCRIPTION file, with `role = c("aut", "cre")`, and make the former maintainer `aut` only.
-* Make sure to change maintainer to your name anywhere else in the package, while retaining the former maintainer as an author (e.g, package level manual file, CONTRIBUTING.md, CITATION, etc.)
-* The [Collaboration Guide](#collaboration) has guidance about adding new maintainers and collaborators
-* Packages that have been archived or [orphaned][] on CRAN don’t need permission of the previous maintainer to be taken over on CRAN. In these cases do get in touch with us so we can offer any help as needed.
```

Comment:
> or [orphaned][] on CRAN

Are we missing a link to somewhere?

Reply:
ouch indeed. we could either remove the link or link to https://cran.r-project.org/web/packages/policies.html :thinking:

Reply:
I think the link is at the end: 

```
[orphaned]: https://cran.r-project.org/src/contrib/Orphaned/README
```
I think this is some other weird way of creating links in markdown? I added that link (using the usual `[]()` notation) in the Spanish translation.

Reply:
thank you! Yes indeed it's a less common way to add links but I'm surprised tinkr missed it. Thanks so much!